examples/ckbtc/test/tests.ts

```diff
@@ -63,26 +63,17 @@ export function getTests(): Test {
 
             const updateBalanceResult = await config.canister.updateBalance();
 
-            if ('Err' in updateBalanceResult) {
-                throw new Error(Object.keys(updateBalanceResult.Err)[0]);
-            }
-
-            expect(true).toBe(true);
+            expect(updateBalanceResult).not.toHaveProperty('Err');
         });
 
         it('fails to update balance for second identity without new utxos', async () => {
             const config = db[1];
 
             const updateBalanceResult = await config.canister.updateBalance();
 
-            if ('Ok' in updateBalanceResult) {
-                throw new Error(
-                    `Expected principal ${config.caller} to not have new UTXOs`
-                );
-            }
-            const errorType = Object.keys(updateBalanceResult.Err)[0];
-
-            expect(errorType).toBe('NoNewUtxos');
+            expect(updateBalanceResult).toEqual(
+                expect.objectContaining({ Err: 'NoNewUtxos' })
```

Comment:
We need to look into the shape of this variant, and look into other of these weird cases to make sure we have things correct